runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -52,20 +52,37 @@ public AbstractQueuedStreamView(final CorfuRuntime runtime,
         super(runtime, streamId, QueuedStreamContext::new);
     }
 
+    /**
+     * Update {@code maxResolution} which is the largest address we
+     * have resolved to so next time we can probably fill from the
+     * {@code resolvedQueue} directly.
+     * <p>
+     * The new {@code maxResolution} may be greater than the actual
+     * tail of this stream, but we know that there is nothing left
+     * to resolve up to that address.
+     *
+     * @param context          stream context
+     * @param newMaxResolution new {@code maxResolution} to set
+     */
+    void updateMaxResolution(QueuedStreamContext context, long newMaxResolution) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651479)